src/utils/createDurationString.test.ts

```diff
@@ -0,0 +1,30 @@
+import createDurationString from './createDurationString';
```

Comment:
how are these tests invoked? Does `npm test` automatically run them?

Reply:
yes, npm run test runs the unit tests for these functions.